File: BAEKJOON/Python/2908.py
a,b = map(str, input().split())
al = []
bl = []
for i in range(3):
    al.append(a[i])
for i in range(3):
    bl.append(b[i])

a = int(al[2])*100 + int(al[1])*10 + int(al[0])
b = int(bl[2])*100 + int(bl[1])*10 + int(bl[0])

if a>b:
    print(a)
elif a<b:
    print(b)

File: BAEKJOON/Java/2908.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String[] input = br.readLine().split(" ");

        StringBuffer sb = new StringBuffer(input[0]);
        int n = Integer.parseInt(sb.reverse().toString());


        sb = new StringBuffer(input[1]);
        int m = Integer.parseInt(sb.reverse().toString());

        if (n > m) {
            System.out.println(n);
        } else {
            System.out.println(m);
        }



    }
}
